packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -614,6 +614,9 @@ export class CallContext {
           if (stream && stream.view) {
             stream.view.scalingMode = scalingMode;
           }
+          if (stream) {
+            stream.selectedScalingMode = scalingMode;
```

Comment:
This works really well, but we are now storing UI logic in the stateful client which I think we have to avoid (cc @PorterNan).
We might have to move tracking inside the video gallery, or (though this would leave react-component users in the dust) moving this state storage to adapter layer

Reply:
Yup, looks it's impacting the behavior coming from sdk so moving this to adapter/composite would be cool, thanks for pointing this out!!

Reply:
I am not sure if we can move the tracking to the composite layer as we are providing this feature at both levels: Composites & Components(Video Gallery & Video Tile). So, if we store the changes in adapter layer and  contoso is building experience using components. The value of scaling mode would be incorrect for them